### ISSUE TITLE
Configure the list form - JSON, ID column needed

### ISSUE THREAD

**Issue by AndrzejKl** [SharePoint/sp-dev-docs#6928]
## Category

- [x] Question
- [ ] Typo
- [ ] Additional article idea

## Question
Hi,
I try to combine in footer link to view item from other list (by ID of this Item) and I always have error "Failure: ID was not found on the data object". I try different naming (ID, id, Id, Identyfikator (polish)) - no success.
Later on I've add extra calculated column (name: "test", formula "=Identyfikator +1") and I see Id's of items. Then I add it to the view, but still I have error "Cannot read property "ValueOf" of null".

Where I've have mistake?

## Environment details (if applicable)

- **Your Developer Environment**: [e.g. Windows 10 | MacOS 10.15.x]
- **Target Environment**: [SharePoint Online]
- **Browser(s)**: [e.g. Chrome v80 | Safari v22 ]

**Comment by artch** [comment]
This indeed looks like a bug to me too, I'm encountering the same behavior which I cannot resolve currently. There is no way to use the `[$ID]` field in the header/footer formatting. In the column formatting it works well.

**Comment by artch** [comment]
@andrewconnell I don't think that the `type:question` label is correct, since it's more of a bug reported by at least two users now.

**Comment by andrewconnell** [comment]
@artch I'd rather leave it to someone to classify something as a bug rather than me... I can't speak on behalf of Microsoft or the SPFx engineering team.
